src/network/netdev/bond.c

```diff
@@ -371,6 +390,82 @@ int config_parse_arp_ip_target_address(const char *unit,
         return 0;
 }
 
+
+int config_parse_ad_user_port_key(const char *unit,
+                                       const char *filename,
+                                       unsigned line,
+                                       const char *section,
+                                       unsigned section_line,
+                                       const char *lvalue,
+                                       int ltype,
+                                       const char *rvalue,
+                                       void *data,
+                                       void *userdata) {
```

Comment:
Please fix the indentation.

Reply:
Will update... Were there already some thoughts regarding a clang-format file?